src/pybind/mgr/rook/module.py

```diff
@@ -257,7 +257,27 @@ def describe_service(self,
         image_name = cl['spec'].get('cephVersion', {}).get('image', None)
         num_nodes = len(self.rook_cluster.get_node_names())
 
+        def sum_running_pods(all_pods: List[Dict[str, Any]], service_type: str, service_name: Optional[str] = None) -> int:
```

Comment:
Given the usage of this function, probably it can be simplified removing the "all_pods" parameter and including inside the command to get the list of all rook pods.

But maybe it can be used to filter other list of pods ... so LGTM.